docs/releases/1.11.txt

```diff
@@ -297,6 +298,9 @@ Database backend API
   support the :lookup:`time` lookup. It accepts a ``field_name`` and ``tzname``
   arguments and returns the SQL necessary to cast a datetime value to time value.
 
+* In order to enable ``FOR UPDATE SKIP LOCKED`` support the
```

Comment:
To enable .... support, set DatabaseFeatures.has_select_for_update_skip_locked = True.